pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,15 +67,20 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
-func applyMemoryCgroupInfoV2(info *SysInfo, controllers map[string]struct{}, _ string) []string {
+func applyMemoryCgroupInfoV2(info *SysInfo, controllers map[string]struct{}, dirPath string) []string {
```

Comment:
dirPath can be like "/sys/fs/cgroup", but we need to check the existence of `/sys/fs/cgroup/${dockerd_cgroup}/memory.swap.max`, not `/sys/fs/cgroup/memory.swap.max`, because `/sys/fs/cgroup/memory.swap.max never appears.

Reply:
You are right, I missed that. I changed the logic a little bit but inside the function.

Reply:
Is there a possibility that `/sys/fs/cgroup/memory.swap.max` will appear? If yes, is it then relevant to check for this use-case?

Reply:
> Is there a possibility that /sys/fs/cgroup/memory.swap.max will appear? 

Yes `/sys/fs/cgroup/memory.swap.max` can appear when the daemon is running inside a cgroup namespace (i.e., dind), but we never need to check `/sys/fs/cgroup/memory.swap.max`